docs/contribute/RFC.md

```diff
@@ -0,0 +1,39 @@
+## Goal
```

Comment:
If this is intended to live on the docs site (I think it should!), then a couple additions are necessary:

1. Add the frontmatter and an H1 (`# RFC process`) to this file
2. Update the table of contents (`/docs/toc.js`) to add a link to this page in the correct section

If this is not intended to live on the docs site, then it should not be in the `/docs` directory. I think it best fits at the root of the repo. Or we could make a `/contributing` directory and move all relevant files in there (and update all references).

Reply:
> (I think it should!)

I agree, IMHO every single bit of non-code should be accessible from the website.

Reply:
Yes I intended for it to live on the docs site! Thanks for the pointers and I'll make the recommended adjustments.